test/functional/services/common/test_subjects.ts

```diff
@@ -72,6 +72,20 @@ export class TestSubjects extends FtrService {
     }
   }
 
+  /**
+   * Get a promise that resolves when an element no longer exists, if the element does exist
+   * it will wait until the element does not exist. If we wait until the timeout and the element
+   * still exists the promise will reject.
+   *
+   * This method is intended to quickly assert that an element does not exist. Its
+   * 2.5 second timeout responds quickly.
+   *
+   * When `options.timeout` is not passed the `timeouts.waitForExists` config is used as
+   * the timeout. The default value for that config is currently 2.5 seconds.
+   *
+   * If the element is hidden but still in the DOM it is treated as "existing", unless `options.allowHidden`
+   * is set to `true`.x-pack/test/functional/services/observability/overview/common.ts
+   */
```

Comment:
Wonderful comment!

Reply:
Wait a sec, should this be: `test/functional/services/common/test_subjects.ts`?
Specifically, should it be: https://github.com/elastic/kibana/blob/main/test/functional/services/common/test_subjects.ts#L16

Reply:
I've added this comment with Spencer when checking the issue and understanding this function. I noticed when I copied it from my previous PR, there is an additional path that was added which I removed. Was that what you mean?

Reply:
Oh I totally like how you documented all of your findings.  I do the same when I'm new to piece of code.

What I'm referring to is this:
>  is set to `true`.x-pack/test/functional/services/observability/overview/common.ts

Is that the correct path, if so where is `allowHidden` mentioned?

 I thought you meant to put [this path](https://github.com/elastic/kibana/blob/main/test/functional/services/common/test_subjects.ts#L16) jn your comment.
 
 I could totally be confused though lol

Reply:
ah cool!